duckduckgo/iOS
# External schemes fix

Task/Issue URL: https://app.asana.com/0/856498667320406/1161274873210818

**Description**:
Some external schemes are not working properly. Examples:
- App Store links.
- Custom External App schemes.

**Steps to test this PR**:

To test:
1. Check App Store/ iTunes links.
2. Check schemes: sms, message, mailto, phone.
3. Ensure general navigation around web works (http, https, ftp, ...).
4. Check iOS-specific ones: facetime, Maps URL.
5. Check some external (custom) schemes.



**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
